docs/developer-docs/latest/development/plugins/email.md

```diff
@@ -117,7 +117,7 @@ module.exports = ({ env }) => ({
   email: {
     provider: 'sendgrid',
     providerOptions: {
-      apiKey: env('SENDGRID_API_KEY'),
+      apiKey: env('SENDGRID_API_KEY', 'YOUR_API_KEY_HERE'),
```

Comment:
Please remove the 2nd parameter, as this acts as a default if the first one isn't defined via a node environment variable or from a .env file.

It's intended that you use a .env file or declare the environment variable not statically set it directly in the configuration.

https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/configurations.html#configuration-using-environment-variables